src/decoder.rs

```diff
@@ -129,20 +157,28 @@ pub fn decode_regular(rsm: RawSourceMap) -> Result<SourceMap> {
 
     let names = rsm.names.unwrap_or_default();
     let sources = rsm.sources.unwrap_or_default();
+    let range_mappings = rsm.range_mappings.unwrap_or_default();
     let mappings = rsm.mappings.unwrap_or_default();
     let allocation_size = mappings.matches(&[',', ';'][..]).count() + 10;
     let mut tokens = Vec::with_capacity(allocation_size);
+    let mut range_tokens = vec![];
 
     let mut nums = Vec::with_capacity(6);
 
-    for (dst_line, line) in mappings.split(';').enumerate() {
+    for (dst_line, (line, rmi_str)) in mappings
+        .split(';')
+        .zip(range_mappings.split(';'))
```

Comment:
There might be less entries in range mappings than in mappings. These should be assumed to be empty